photo/models.py

```diff
@@ -1,3 +1,101 @@
+from datetime import timezone
+
+from django.contrib.auth.models import AbstractBaseUser, PermissionsMixin
 from django.db import models
 
+
 # Create your models here.
+class User(AbstractBaseUser, PermissionsMixin):
+    USERNAME_FIELD = "email"
+
+    email = models.CharField(max_length=255, primary_key=True)
+    name_first = models.CharField(max_length=255, blank=True, null=True)
+    name_last = models.CharField(max_length=255, blank=True, null=True)
+    profile_picture = models.ForeignKey(
+        "Picture",
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        db_column="picture_id",
+    )
+    profile_picture_updated_at = models.DateTimeField(blank=True, null=True)
+    user_handle = models.CharField(max_length=255, blank=True, null=True)
+
+
+class Picture(models.Model):
+    user = models.ForeignKey(
+        "User",
+        on_delete=models.CASCADE,
+        db_column="user_id",
+    )
+    picture_url = models.CharField(max_length=255, primary_key=True)
+    likes = models.ManyToManyField(User, blank=True, null=True)
+
+
+class PictureComment(models.Model):
+    user = models.ForeignKey(
+        "User",
+        on_delete=models.CASCADE,
+        db_column="user_id",
+    )
+    picture = models.ForeignKey(
+        "Picture",
+        on_delete=models.CASCADE,
+        db_column="picture_id",
+    )
+    text = models.TextField(blank=False, null=False)
+    created_at = models.DateTimeField(default=timezone.now)
+
+
+class Collection(models.Model):
+    name = models.CharField(max_length=255, blank=True, null=True)
+    user = models.ForeignKey(
+        "User",
+        on_delete=models.CASCADE,
+        db_column="user_id",
+    )
+    pictures = models.ManyToManyField(Picture, blank=True, null=True)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["name", "user"], name="collection_pk")
+        ]
+
+
+class Contest(models.Model):
+    title = models.CharField(max_length=255, blank=True, null=True)
+    description = models.TextField(blank=False, null=False)
+    cover_picture = models.ForeignKey(
+        "Picture",
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        db_column="picture_id",
+    )
+    prize = models.CharField(max_length=255, null=True)
+    automated_dates = models.BooleanField(default=True)
+    upload_phase_start = models.DateTimeField(default=timezone.now)
+    upload_phase_end = models.DateTimeField(null=True)
+    voting_phase_end = models.DateTimeField(null=True)
+    active = models.BooleanField(default=True)
+    winners = models.ManyToManyField(User, blank=True, null=True)
+    created_by = models.ForeignKey(
+        "User",
+        on_delete=models.CASCADE,
+        db_column="user_id",
+    )
+
+
+class ContestSubmission(models.Model):
+    user = models.ForeignKey(
+        "User",
+        on_delete=models.CASCADE,
+        db_column="user_id",
+    )
```

Comment:
we should only need the picture. My understanding is that a user submits a new picture for the contest, so we create a new picture object and add it to the submission. We know that it was submited by that user because the picture has a user field